packages/calypso-e2e/src/lib/flows/login-flow.ts

```diff
@@ -14,14 +22,19 @@ export class LoginFlow {
 	 * Creates an instance of the log in flow.
 	 *
 	 * @param {Page} page Object representing the base page.
-	 * @param {string} [accountType] Type of account to be used for the log in process.
+	 * @param {string} [account] Account information to be used to login. Can be either an account type from config, or credentials.
```

Comment:
Just a heads up that I am looking to refactor LoginFlow and LoginPage to hopefully eliminate one of them - do you think they should be kept?

Reply:
and by kept I mean the separation of concerns between `LoginPage` and `LoginFlow.` The more I work around them I realize it seems a bit convoluted to have two pieces.

Reply:
Oh no, they can definitely be squashed together, and that's a fantastic idea! But, I think we need to keep the behavior added here when we do, namely...

1. The ability to provide creds or an account type (or maybe just creds, and make the test look up the account type if they want to use that)
2. The ability to add an expected landing page to wait for navigation post-login.

Reply:
> The ability to provide creds or an account type

Yep, the base `LoginPage.login` method already accepts an `username/password` combination. I like the automatic lookup of `testAccountCredentials` that LoginFlow provides, but no longer think it should have a default value (`defaultUser`) like it does now. 

As I've worked on this project I've come to think anything involving data should be more explicit, so having the test step explicitly obtain the credentials would be in line with that. Less of the wishy-washy defaults that plagued the Selenium suite.